app/src/main/java/org/wikipedia/talk/TalkTopicsActivity.kt

```diff
@@ -247,6 +247,7 @@ class TalkTopicsActivity : BaseActivity() {
 
     companion object {
         private const val EXTRA_PAGE_TITLE = "pageTitle"
+        const val NEW_TOPIC_ID = -2
```

Comment:
Lgtm .. just a minor q... is this a randomly picked value we are giving to a new topic?

Reply:
Yes. Please let me know if you prefer another number.

Reply:
oh ok got it... No .. Lgtm..